README.md

```diff
@@ -1,57 +1,34 @@
-# Project Name
-
-(short, 1-3 sentenced, description of the project)
-
-## Features
-
-This project framework provides the following features:
-
-* Feature 1
-* Feature 2
-* ...
+# Azure Communication Network Traversal sample
 
 ## Getting Started
 
 ### Prerequisites
 
-(ideally very short, if any)
-
-- OS
-- Library version
-- ...
+- You need Node.js
+- Firefox, Edge or Chrome
+- Computer with video camera
+- A Communication Services resource to get the Connection String.
 
 ### Installation
 
-(ideally very short)
-
-- npm install [package name]
-- mvn install
-- ...
+- In an Administrator Powershell go to the sample dir and run the following
```

Comment:
npm shouldn't need admin? can you verify whether this actually has to be elevated or not?